src/ServicePulse.Host/vue/src/views/EventsView.vue

```diff
@@ -0,0 +1,28 @@
+<script setup>
+import { licenseStatus } from "../composables/serviceLicense.js";
+import LicenseExpired from "../components/LicenseExpired.vue";
+import DataView from "../components/DataView.vue";
+import EventLogItem from "../components/EventLogItem.vue";
+</script>
+
+<template>
+  <LicenseExpired />
+  <template v-if="!licenseStatus.isExpired">
+    <div class="events events-view">
+      <DataView api-url="eventlogitems" :auto-refresh="5000" :items-per-page-options="[20, 35, 50, 75]" :items-per-page="20">
+        <template #header><h1>Events</h1></template>
+        <template #item="item">
+          <EventLogItem :eventLogItem="item" />
+        </template>
+      </DataView>
+    </div>
+  </template>
+</template>
+
+<style>
+/* TODO: Fix up styles */
```

Comment:
what's left here?

Reply:
I need to go through all of the styles and see what is appropriate to put in each location. It's a bit of a mess at the moment.

Reply:
Done